static_frame/core/index_hierarchy.py

```diff
@@ -1603,184 +2305,209 @@ def loc_searchsorted(self,
 
         length = self.__len__()
         if sel.ndim == 0 and sel == length: # an element:
-            return fill_value #type: ignore [no-any-return]
+            return fill_value
 
         flat = self.flat().values
         mask = sel == length
         if not mask.any():
-            return flat[sel] #type: ignore [no-any-return]
+            return flat[sel]
 
         post = np.empty(len(sel), dtype=object)
         sel[mask] = 0 # set out of range values to zero
         post[:] = flat[sel]
         post[mask] = fill_value
         post.flags.writeable = False
-        return post #type: ignore [no-any-return]
-
-
-
+        return post
 
-    #---------------------------------------------------------------------------
+    # --------------------------------------------------------------------------
     # export
 
-    def _to_frame(self,
-            constructor: tp.Type['Frame']
+    def _to_frame(self: IH,
+            constructor: tp.Type['Frame'],
             ) -> 'Frame':
-
         if self._recache:
-            self._update_array_cache()
+            self._update_array_cache() # pragma: no cover
```

Comment:
Same here...

Reply:
See this response: https://github.com/InvestmentSystems/static-frame/pull/437#discussion_r831501537